config/express.server.config.ts

```diff
@@ -21,11 +22,13 @@ export default function () {
   // This is required for parsing application/json in req.body
   app.use(bodyParser.json());
   app.use(cors());
-  app.use(
-    pinoHttp({
-      logger,
-    }),
-  );
+
+  // TODO: comment this back in when done
+  // app.use(
+  //   pinoHttp({
+  //     logger,
+  //   }),
+  // );
```

Comment:
why was this commented out?

Reply:
couldn't see test results at a glance and scrolling in tmux requires a keyboard command. I'll put it back in.

Reply:
I'm happy to approve once that is back in

Reply:
you might want to set the logger verbosity different on tests